### PR TITLE
has_and_belong_to_many association with teacher and  course

### PR DESCRIPTION
**Pull request by Gullu143**

### CODE REVIEW

File: app/controllers/teachers_controller.rb [hunk 1]
```diff
@@ -4,6 +4,11 @@ def index
   	@teacher = Teacher.all
   end
 
+  def teacher_course
```
Comment (saqib1324) [inline app/controllers/teachers_controller.rb]: this should be plural as this method returns courses not a single course, this method name should be `teacher_courses`

File: app/views/teachers/teacher_course.html.erb [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<% @courses.each do |course| %>
```
Comment (saqib1324) [inline app/views/teachers/teacher_course.html.erb]: change this file_name to `teacher_courses.html.erb` as we are changing the method name

File: config/routes.rb [hunk 3]
```diff
@@ -3,8 +3,10 @@
   devise_for :teachers, controllers: { sessions: 'teachers/sessions', registrations: 'teachers/registrations' }
   post '/create_new_teacher', :to => 'teachers#create_new_teacher'
   resources :teachers
+  get 'teacher_course', to: 'teachers#teacher_course'
```
Comment (saqib1324) [inline config/routes.rb]: change the url to `/my-courses` as this url retuns the courses for current_teacher, thus
the line will be
`get 'my_courses', to: 'teachers#teacher_courses'

File: app/controllers/teachers_controller.rb (hunk 1 above)
Comment (Gullu143) [reply app/controllers/teachers_controller.rb]: ok. I make it plural

**Pull request merged**